web/src/pages/Dashboard/index.tsx

```diff
@@ -32,27 +112,34 @@ export default function Dashboard() {
           <div id="IPCA">
             <h1>Títulos indexados ao IPCA</h1>
             <div id="card-list">
-              <Card />
-              <Card />
-              <Card />
-              <Card />
-              <Card />
-              <Card />
+              {bonds &&
+                [...bonds.IPCA, ...bonds.IGPN].map(bond => (
+                  <Card key={bond.id} {...bond} />
+                ))}
             </div>
           </div>
           <div id="fixed-rate">
             <h1>Títulos pré-fixados</h1>
             <div id="card-list">
-              <Card />
-              <Card />
-              <Card />
-              <Card />
+              {bonds?.PREFIXADO.map(bond => (
+                <Card key={bond.id} {...bond} />
+              ))}
             </div>
           </div>
           <div id="postfixed">
             <h1>Títulos pós-fixados</h1>
             <div id="card-list">
-              <Card />
+              {bonds?.SELIC.map(bond => (
+                <Card key={bond.id} {...bond} />
+              ))}
+            </div>
+          </div>
+          <div id="IGPN">
```

Comment:
"Emitidos somente até 2006, esses títulos têm vencimento marcado para o primeiro dia do ano de 2031."
I'd say we can safely ignore IGP-M bonds as they aren't circulating in the market anymore. They can only be "leveraged" through investment funds that have purchased them long ago. I don't think they'll come back as IPCA is a much more common inflation index nowadays.
In short, don't list or create a section for IGP-M bonds